src/commands/start.ts

```diff
@@ -204,7 +205,7 @@ export default class Start extends Command {
         case 'subquery':
           return ['subquery'];
         case 'gql':
-          return ['tooling'];
+          return ['tooling', 'graphql'];
```

Comment:
I think this should go with `subquery`

Reply:
I moved it to be only in `subquery`. I think thats the better tag name.

It seems to be OK being in services array twice. I don't want to dedupe for no reason, but I also don't want there to be some issue with `-o gql -o tooling` either. I think not making an exception is safer.